core/src/main/java/org/apache/accumulo/core/fate/FateExecutor.java

```diff
@@ -205,14 +205,14 @@ protected Set<Fate.FateOperation> getFateOps() {
    * Initiates the shutdown of this FateExecutor. This means the pool executing TransactionRunners
    * will no longer accept new TransactionRunners, the currently running TransactionRunners will
    * terminate after they are done with their current transaction, if applicable, and the work
-   * finder is interrupted. {@link #isShutdown()} returns true after this is called.
+   * finder is shutdown. {@link #isShutdown()} returns true after this is called.
    */
   protected void initiateShutdown() {
     transactionExecutor.shutdown();
     synchronized (runningTxRunners) {
       runningTxRunners.forEach(TransactionRunner::flagStop);
     }
-    workFinder.interrupt();
```

Comment:
Interrupting  threads can be useful for the case when they are stuck on IO and not checking the boolean.  Was this interrupt causing a problem?  It does seem inconsistent, we at not interrupting other threads, so removing the interrupt makes it consistent.